packages/core/src/components/Table/TableCell/TableCell.tsx

```diff
@@ -125,42 +131,25 @@ export const HvTableCell = forwardRef<HTMLElement, HvTableCellProps>(
       ...others
     } = props;
     const { classes, cx } = useClasses(classesProp);
-    const { activeTheme, selectedMode } = useTheme();
+    const { selectedMode } = useTheme();
     const tableContext = useContext(TableContext);
     const tableSectionContext = useContext(TableSectionContext);
 
-    const [sortedColorValue, setSortedColorValue] = useState<
-      string | undefined
-    >();
-    const [sortedColorAlpha, setSortedColorAlpha] = useState<
-      string | undefined
-    >();
-
     const type = typeProp || tableSectionContext?.type || "body";
 
     const Component =
       component || tableContext?.components?.Td || defaultComponent;
 
     const TableCell = useMemo(() => StyledTableCell(Component), [Component]);
 
-    let sortedColor =
-      checkValidHexColorValue(sortedColorValue) && sortedColorAlpha
-        ? hexToRgbA(sortedColorValue, parseFloat(sortedColorAlpha))
-        : sortedColorValue;
+    const sortedColor = useRef<string>();
```

Comment:
This seems to not be updating correctly:

https://github.com/lumada-design/hv-uikit-react/assets/43220251/ad9e1765-cbf4-4206-b334-7bce4ff18e75

Potential solution:

```suggestion
     const [sortedColor, setSortedColor] = useState<string | undefined>();

    useEffect(() => {
      const getColor = (value?: string) => {
        if (value?.includes("var(--uikit-colors-")) {
          const color = value
            .replace("var(--uikit-colors-", "")
            .replace(")", "")
            .trim();

          return activeTheme?.colors.modes[selectedMode][color];
        } else {
          return value;
        }
      };

      const color = getColor(activeTheme?.table.rowSortedColor);

      setSortedColor(
        getSortedColor(color, activeTheme?.table.rowSortedColorAlpha)
      );
    }, [selectedMode, activeTheme]);
```